pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -175,13 +315,26 @@ def preprocess(
         * Does not support finite shots.
         * Does not support derivatives.
         * Does not support vector-Jacobian products.
+        * Does not support batch processing.
         """
 
         # TODO: remove comments when cuTensorNet MPS backend is available as a prototype
-        # config = self._setup_execution_config(execution_config)
-        # program = self._interface.preprocess()
-        # return program, config
+        config = self._setup_execution_config(execution_config)
+
+        program = TransformProgram()
+
+        program.add_transform(validate_measurements, name=self.name)
+        program.add_transform(validate_observables, accepted_observables, name=self.name)
+        program.add_transform(validate_device_wires, self._wires, name=self.name)
+        program.add_transform(
```

Comment:
Have you tested the decomposition rules with multi-qubit gates?

Reply:
yes, `Toffoli`, `OrbitalRotation`...but `DoubleExcitationPlus` doesn't work.